mne/io/bti/bti.py

```diff
@@ -1134,11 +1136,14 @@ def _get_bti_info(pdf_fname, config_fname, head_shape_fname, rotation_x,
 
     use_hpi = False  # hard coded, but marked as later option.
     logger.info('Creating Neuromag info structure ...')
-    info = _empty_info()
+    if 'sample_period' in bti_info.keys():
+        sfreq = 1e3 / bti_info['sample_period'] * 1e-3
```

Comment:
Just noticed this statement does a bit of silly calculation. Should juest be `sfreq = 1. / bti_info['sample_period']`